humanized_opening_hours/locales/pt/LC_MESSAGES/hoh.pot

```diff
@@ -0,0 +1,168 @@
+# Brazilian portuguese translation for HOH.
+# Copyright (C) 2018
+# GABRIEL LUIZ FREITAS ALMEIDA gabrielf.almeida90@gmail.com, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-10-06 10:59-0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: GABRIEL LUIZ FREITAS ALMEIDA gabrielf.almeida90@gmail.com\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: pt-br\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: temporal_objects.py:210
+msgid "Open 24 hours a day and 7 days a week."
+msgstr "Aberto 24 horas por dia e 7 dias por semana"
```

Comment:
As this sentence is used "as is" by the script, can you add a final point?

Thank you for your work! :)